cfgov/v1/models/home_page.py

```diff
@@ -23,19 +23,9 @@ class HomePage(CFGOVPage):
         ('half_width_link_blob', molecules.HalfWidthLinkBlob()),
     ], blank=True)
 
-    latest_updates = StreamField([
-        ('posts', blocks.ListBlock(blocks.StructBlock([
-            ('categories', blocks.ChoiceBlock(choices=ref.limited_categories,
-                                              required=False)),
-            ('link', atoms.Hyperlink()),
```

Comment:
I think removing this orphans an `import atoms` and an `import blocks` above.

Reply:
Doh!

Reply:
Removed in 9e8c6c7.